Formula/msmtp.rb

```diff
@@ -1,8 +1,8 @@
 class Msmtp < Formula
   desc "SMTP client that can be used as an SMTP plugin for Mutt"
   homepage "http://msmtp.sourceforge.net"
-  url "https://downloads.sourceforge.net/project/msmtp/msmtp/1.6.4/msmtp-1.6.4.tar.xz"
-  sha256 "9b49c022a5440d41b6782c97ef2977d0346c3dae05aa8836243a9953e982d1cd"
+  url "https://downloads.sourceforge.net/project/msmtp/msmtp/1.6.4/msmtp-1.6.5.tar.xz"
```

Comment:
Can't seem to download this: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/2594/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/fetch_msmtp/